Styling/README.md

```diff
@@ -46,6 +46,7 @@ Contents:
    * [Variables](#variables)
    * [Built-in Variables](#built-in-variables)
    * [Built-in Functions](#built-in-functions)
+   * [Batch Table Hierarchy](#batch-table-hierarchy)
```

Comment:
It fits ok, as those sections describe expressions related to the hierarchy and point clouds. I'm fine with placing them in a different section too.